Gemfile

```diff
@@ -2,7 +2,7 @@
 
 source 'https://rubygems.org'
 
-ruby '3.0.3'
+ruby '3.1.0'
```

Comment:
pls don't change the specified ruby version, the PR must contain only the HW2 code